typescript/sdk/src/timelock/evm/EvmTimelockDeployer.hardhat-test.ts

```diff
@@ -164,4 +171,187 @@ describe('EvmTimelockDeployer', async () => {
 
     expect(await timelock.getMinDelay()).to.equal(0);
   });
+
+  it('should set anyone as a executor if no one is set in the input config', async () => {
+    const zeroDelayConfig: TimelockConfig = {
+      minimumDelay: 0,
+      proposers: [signer.address],
+      admin: signer.address,
+    };
+
+    const { TimelockController } = await deployer.deployContracts(
+      TestChainName.test4,
+      zeroDelayConfig,
+    );
+    const timelockAddress = TimelockController.address;
+
+    const timelock = TimelockController__factory.connect(
+      timelockAddress,
+      multiProvider.getProvider(TestChainName.test4),
+    );
+
+    expect(
+      await timelock.hasRole(EXECUTOR_ROLE, hre.ethers.constants.AddressZero),
+    ).to.be.true;
+
+    const _0_BYTES_32 =
```

Comment:
Why is this calldata construction different from everywhere else? Imo for unit testing, you want to only change the one thing that is necessary for the test (which in this case is no executor in the config).